Automattic/wp-calypso
# Try extracting load-script to a new package

#### Changes proposed in this Pull Request

This extracts `load-script` to a new package, which will make it easier to reuse other modules as packages (specifically Analytics).

#### Testing instructions

* Check that sections of Calypso that use `load-script` still work:
- /log-in
- /checkout
- location search
- /stats
- embeds
- facebook button
- analytics
- directly
- translator jumpstart
- geocoding
- videopress preview
